### ISSUE TITLE
*_rect

### ISSUE THREAD

**Issue by DanGrayson** [UniMath/UniMath#580]
See the comment at https://github.com/UniMath/UniMath/pull/567#discussion_r101888918

Rename induction principles such as UU_rect to UU_rec, after #567 gets merged.

Also think about making interfaces to them more uniform.

Also add notations, such as total2_rec for total2_rect, etc.

**Comment by benediktahrens** [comment]
On 02/18/2017 03:16 PM, Daniel R. Grayson wrote:
> See the comment at https://github.com/UniMath/UniMath/pull/567#discussion_r101888918
> 
> Rename induction principles such as UU_rect to UU_rec, after #567 gets merged.
> 
> Also think about making interfaces to them more uniform.
> 
> Also add notations, such as total2_rec for total2_rect, etc.

Also, upstream the induction principle for isomorphisms in a univalent
category from UniMath/TypeTheory, file Auxiliary/Auxiliary.v.

**Comment by DanGrayson** [comment]
Yes, if we introduce notations for the recursion schemes that end with "rec", we can rename those identifiers of ours then, too.
